examples/b2c/checkout/integrations/classes/B2CDeliverySample.cls

```diff
@@ -232,10 +232,10 @@ global class B2CDeliverySample implements sfdc_checkout.CartShippingCharges {
         // ReferenceNumber: Reference Number from External Service
         // IsActive: If this Option is Active
         Id productId = getDefaultShippingChargeProduct2Id();
-        //String cartCurrency = [SELECT CurrencyIsoCode FROM WebCart WHERE Id = :webCartId][0].CurrencyIsoCode;
+        String cartCurrency = [SELECT CurrencyIsoCode FROM CartDeliveryGroup WHERE Id = :cartDeliveryGroupId][0].CurrencyIsoCode;
         CartDeliveryGroupMethod cartDeliveryGroupMethod = new CartDeliveryGroupMethod(
             CartDeliveryGroupId = cartDeliveryGroupId,
-            //CurrencyIsoCode = cartCurrency,
+            CurrencyIsoCode = cartCurrency,
```

Comment:
@snilakandan13 does cart always have a default currency? or is there any where this field is not populated for cart?

Reply:
in the cartDeliveryGroup, this will always be there